src/components/elementView/elementTab.tsx

```diff
@@ -318,11 +318,22 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
   };
 
   private createUser = (user: CreateUserObject, headers: any): Promise<any> => {
-    return axios.post(API_ROOT + "api/users/add", user, headers).then((res) => {
-      this.handleDataUpdate(true);
-      this.handleClose();
-      this.addSuccessToast(res.data.success_message);
-    });
+    const username = user.username;
+    return axios
+      .post(API_ROOT + "api/users/add", user, headers)
+      .then((res) => {
+        this.handleDataUpdate(true);
+        this.handleClose();
+        this.addSuccessToast("SUCCESS: User " + username + " created.");
+      })
+      .catch((err) => {
+        for (let key in err.response.data) {
+          let errors = err.response.data[key];
+          errors.forEach((message: string) => {
+            this.addErrorToast(message);
+          });
```

Comment:
the errors weren't being caught as a toast before which explains why the 403 HTML page was showing up one character at a time. since the errors at this endpoint are thrown by `django-rest-auth` and not our code, we can't show `failure_message` as usual, so i'm parsing the list of errors that gets returned instead